matthewpoer/SuiteCRM
# Fix #5701 with SugarTheme Enhancement

Custom CSS generated for the SuiteP theme was being ignored due to the subTheme system. This enhancement resolves by self-referencing for the current "primary" theme, then descending to get any files for the subTheme.

## Description
Existing customizations from prior versions of Sugar/Suite are using this architecture and it should continue to be supported, while also supporting extra CSS for the Dawn/Dark/etc. subThemes.

Issue logged as https://github.com/salesagility/SuiteCRM/issues/5701

## How To Test This
Install a custom CSS file in `custom/themes/SuiteP/css/style.css` that does something obvious, e.g. 

~~~
h2.module-title-text {
  font-weight: bold;
  color: pink;
}
~~~

QRR and notice that the change isn't seen. Apply the patch and the change will appear because this file is found and used.

## Types of changes

### Final checklist
